daemon/create.go

```diff
@@ -44,6 +44,11 @@ func (daemon *Daemon) ContainerCreate(job *engine.Job) error {
 		return fmt.Errorf("You should always set the Memory limit when using Memoryswap limit, see usage.\n")
 	}
 
+	if !daemon.SystemConfig().OomKillDisable {
+		job.Errorf("Your kernel does not support oom kill disable.\n")
```

Comment:
I think that writes to the job's stderr for later use (handling error after returning from job.Run)